### PR TITLE
Add endpoint to get S3 POST credentials

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@k88hudson r?

File: server/routes/image-request.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    var user = req.session && req.session.user;
+
+    // Check auth
+    if (habitat.get('FAKE_AUTH')) {
+        user = req.body.user;
+    } else {
+        if (!user) return next(errorUtil(401, 'No user session found'));
+        if (!user.id || !user.username) return next(errorUtil(401, 'No valid user session found'));
+    }
+console.log(user);
```
Comment (k88hudson) [inline server/routes/image-request.js]: stray console.log

**Comment by k88hudson** [comment]
looks good R+

**Comment by k88hudson** [comment]
except that one nit

**Pull request merged**